opentelemetry-api/opentelemetry/trace/__init__.py

```diff
@@ -84,15 +89,19 @@ def get_current_span(self) -> 'Span':
             invalid :class:`.SpanContext`.
         """
 
-
     @contextmanager
-    def start_span(self, name: str, parent: 'Span') -> Iterator['Span']:
+    def start_span(self,
+                   name: str,
+                   parent: 'Span' = CURRENT_SPAN) -> Iterator['Span']:
```

Comment:
Nit: Normally you don't put spaces around the `=` in a default argument, but I don't know how the situation is with type annotations.

Reply:
Same, was not sure about how things work with annotations (had barely used them before ;) ) 

Hope we can get a linter enabled soon :)

Reply:
This is right! And pylint actually does report on this if you remove the spaces:

```
opentelemetry/trace/__init__.py:95:33: C0326: Exactly one space required around keyword argument assignment
                   parent: 'Span'=CURRENT_SPAN) -> Iterator['Span']:
                                 ^ (bad-whitespace)
```